Remove these definitions because they take precedence over variants
which can then not be selected.  (closes: #347496)

Index: xc/programs/xkbcomp/rules/xorg
===================================================================
--- xc/programs/xkbcomp/rules/xorg.orig	2006-02-05 21:59:24.000000000 -0500
+++ xc/programs/xkbcomp/rules/xorg	2006-02-05 21:59:26.000000000 -0500
@@ -314,8 +314,6 @@
  $pcmodels      intl   			=       us(%mcompose)
   *		intl			=	us(pc104compose)
   *		jp			=	pc/pc(pc102)+pc/jp(latin)+pc/jp:2
- $pcmodels      ca     			=       pc/pc(%m)+pc/ca(multi)+pc/ca(multi-2gr):2+group(rctrl_switch)
-  *             ca     			=       pc/pc(pc105)+pc/ca(multi)+pc/ca(multi-2gr):2+group(rctrl_switch)
  $pcmodels $nonlatin			=	pc/pc(%m)+pc/us+pc/%l%(v):2
  $pcmodels	*			=	pc/pc(%m)+pc/%l%(v)
   *	   $nonlatin			=	pc/pc(pc105)+pc/us+pc/%l%(v):2
